### ISSUE TITLE
Captured screenshot should be visible

### ISSUE THREAD

**Issue by sguptatw** [getgauge/html-report#164]
**Expected behavior**
Captured screenshot should be visible clearly on the html-report

**Actual behavior**
<img width="631" alt="smallscreenshot" src="https://user-images.githubusercontent.com/6310197/34554084-1a212190-f151-11e7-9c41-ff9e157358b7.png">

**Steps to replicate**
* Have a step failure in gauge-js project.

**Version**
```
Gauge version: 0.9.7.nightly-2018-01-02
Commit Hash: 0a6907a

Plugins
-------
html-report (4.0.3.nightly-2017-12-08)
js (2.1.0.nightly-2018-01-02)
```

**Comment by osandadeshan** [comment]
@negiDharmendra 
I think the screenshot should be placed on right side of the screen and it should be much larger than this. In previous versions, screenshots were in the ideal place with the ideal sizes.

**Comment by negiDharmendra** [comment]
@osandadeshan @sguptatw Thoughts on the fix.
Screenshot with failure and multiple custom screenshot.

<img width="1631" alt="screen shot 2018-12-07 at 12 09 19 pm" src="https://user-images.githubusercontent.com/15261737/49632238-003f2780-fa1b-11e8-8df5-41adf45281a0.png">
<img width="1667" alt="screen shot 2018-12-07 at 12 09 50 pm" src="https://user-images.githubusercontent.com/15261737/49632251-0cc38000-fa1b-11e8-90f4-20dd69323757.png">

**Comment by osandadeshan** [comment]
@negiDharmendra 
This is great

**Comment by sguptatw** [comment]
[Blocker] The latest nightly available is 4.0.7.nightly-2018-12-06

**Comment by negiDharmendra** [comment]
The fix should be available in nightly >= 2018-12-10
